web/auth/oauth.go

```diff
@@ -352,6 +353,12 @@ func authorize(c echo.Context) error {
 		}
 	}
 
+	// Fill the client_os of the OAuth client
+	rawUserAgent := c.Request().UserAgent()
+	ua := user_agent.New(rawUserAgent)
+	params.client.ClientOS = ua.OS()
+	_ = couchdb.UpdateDoc(instance, params.client)
```

Comment:
Out of curiosity, why the `_` ? Is it required to get the returned value?

Reply:
In go, nope, but our linter is not happy when a function returns an error and we don't use it. Assigning the error to `_` is a way to tell to the linter that it is OK.